crates/api_structs/src/site.rs

```diff
@@ -94,17 +98,17 @@ pub struct SiteResponse {
 #[derive(Serialize)]
 pub struct GetSiteResponse {
   pub site_view: Option<SiteView>, // Because the site might not be set up yet
-  pub admins: Vec<UserViewSafe>,
-  pub banned: Vec<UserViewSafe>,
+  pub admins: Vec<PersonViewSafe>,
+  pub banned: Vec<PersonViewSafe>,
   pub online: usize,
   pub version: String,
-  pub my_user: Option<UserSafeSettings>,
+  pub my_user: Option<LocalUserSettingsView>,
```

Comment:
One of the bigger changes. `my_user` is now a view which contains `PersonSafe`, `PersonAggregates` and `LocalUserSettings`

Reply:
Almost sounds like we should make an API v3 for this. At this rate, we will reach v10 within a year.

Reply:
After I get #1493 pulled in, I def think we should go to an API v3. Its enough major changes for it anyway.